closed/src/java.base/share/classes/com/sun/crypto/provider/NativeGaloisCounterMode.java

```diff
@@ -343,40 +357,48 @@ int encrypt(byte[] in, int inOfs, int len) {
      */
     int encryptFinal(byte[] in, int inOfs, int len, byte[] out, int outOfs)
         throws IllegalBlockSizeException, ShortBufferException {
+        byte[] keyCopy;
+        byte[] ivCopy;
+        byte[] aad;
+        int tagLenBytes = this.tagLenBytes;
```

Comment:
This needs to be in the `synchronized` block; please don't reuse the field name.

Reply:
Added them to the synchronized block and renamed the local `tagLenBytes`.